Detectors/EMCAL/base/include/EMCALBase/Point.h

```diff
@@ -0,0 +1,98 @@
+#ifndef ALICEO2_EMCAL_POINT_H
+#define ALICEO2_EMCAL_POINT_H
+
+#include <FairMCPoint.h>
+
+namespace AliceO2 {
+  namespace EMCAL {
+    
+    /// \class Point
+    /// \brief EMCAL simulation point information
+    class Point : public FairMCPoint {
+    public:
```

Comment:
@sawenzel OK, I will have a look and try to incorporate the changes into the pull request.